website/src/documentation/patterns/ai-platform.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: section
+title: "AI Platform integration patterns"
+section_menu: section-menu/documentation.html
+permalink: /documentation/patterns/ai-platform/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AI Platform integration patterns
+
+This page describes common patterns in pipelines with Google Cloud AI Platform transforms.
+
+<nav class="language-switcher">
+  <strong>Adapt for:</strong>
+  <ul>
+    <li data-type="language-java">Java SDK</li>
+    <li data-type="language-py" class="active">Python SDK</li>
+  </ul>
+</nav>
+
+## Getting predictions
+
+This section shows how to use [Google Cloud AI Platform Prediction](https://cloud.google.com/ai-platform/prediction/docs/overview) to make predictions about new data from a cloud-hosted machine learning model.
+ 
+[tfx_bsl](https://github.com/tensorflow/tfx-bsl) is a library with a Beam PTransform called `RunInference`. `RunInference` is able to perform an inference that can use a service endpoint. When using a service endpoint, the transform takes a PCollection of type `tf.train.Example` and, for every bunch of elements, sends a request to AI Platform Prediction. The transform produces a PCollection of type `PredictLog`, which contains predictions.
```

Comment:
"for every bunch of elements, sends a request to AI Platform Prediction."

I assume this refers to batching data to AI Platform. Is there away to configure how many elements are in the batch? I think the previous version of this paragraph stated that one request is sent per element.

Reply:
>  Is there away to configure how many elements are in the batch?

No, in this case Beam handles the size of a batch on its own.

> I think the previous version of this paragraph stated that one request is sent per element.

That's right. After a discussion on batching it comes to my mind this is not truly correct.

Reply:
Is there another doc (or some source code) we can link to that describes how Beam handles the batch size? It's probably not worth adding the extra details to this pattern but it may be helpful to provide a link for more information.

I also think we should use the term "batch" instead of "bunch" for additional clarity.

Reply:
I believe BatchElements (the transform responsible for batching here) doc is sufficient. Here's a [link](https://github.com/apache/beam/blob/42d79c29949725b3afabfb3754bfb394be594460/sdks/python/apache_beam/transforms/util.py#L540)

Reply:
Nice. We should add the link to the page